SCClassLibrary/Common/Streams/TimePatterns.sc

```diff
@@ -4,7 +4,10 @@ Pstep : Pattern {
 	*new { arg levels, durs = 1, repeats = 1;
 		^super.newCopyArgs(levels, durs, repeats).init
 	}
-	init { if (list.isKindOf(Collection)) { list = Pseq(list); } }
+	init {
+		if (list.isKindOf(Collection)) { list = Pseq(list); };
```

Comment:
instead of `list.isKindOf(Collection)` it should be `list.isSequenceableCollection`. A `Set`, e.g. won't work as an argument to `Pseq`.